integration/actions_test.go

```diff
@@ -262,9 +262,22 @@ func queryRow(database string, query string, expected ...interface{}) action {
 }
 
 // Create a database for use by a module.
-func createDB(t testing.TB, module, dbName string) {
+func createDBAction(module, dbName string, isTest bool) action {
+	return func(t testing.TB, ic testContext) error {
+		createDB(t, module, dbName, isTest)
+		return nil
+	}
+}
+
+func createDB(t testing.TB, module, dbName string, isTestDb bool) {
```

Comment:
Rather than burdening this function with the flag on every invocation, how about we just append `_test` to the small number of invocations that require it?

Reply:
I'd started with that approach but the issue was that then the envar value included the _test suffix, then the ftltest code appended the suffix again (dbname_test_test).
So this func needs to put the untouched value to the envar, and use the suffixed value in all the postgres steps

Reply:
makes sense